core/iwasm/aot/aot_loader.c

```diff
@@ -2182,9 +2185,12 @@ load_import_globals(const uint8 **p_buf, const uint8 *buf_end,
                 tmp_global.global_data_linked;
             import_globals[i].is_linked = true;
         }
-#else
-        import_globals[i].is_linked = false;
+        else
 #endif
+#endif /* WASM_ENABLE_MULTI_MODULE != 0 */
```

Comment:
When multi-module isn't enabled and libc-builtin is enabled, the global isn't linked by default, is it expected? Or will the iwasm's main.c link the global when instantiating?

Reply:
YES. only apply `wasm_native_lookup_libc_builtin_global()` when executing loading linking. Otherwise, `core/iwasm/libraries/libc-builtin/spec_test_builtin_wrapper.c` will take over